From 44164fa8b413a1a3d3f3b92c36aeedded314d8d9 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Thu, 18 Apr 2019 20:14:13 +0300
Subject: [PATCH] [SW-0] i2c: Gaudi update CPLD location

The wrong address cause sensors to hang
Now values are N/A but at least no hang

Change-Id: I544143cb5458995b35adc837f533e5d402023cf3
TODO: figure out real address
---

diff --git a/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat b/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
index f20aacc..438a0fa 100755
--- a/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
+++ b/board/habanalabs/common/rootfs_overlay/etc/init.d/S90i2c-instantiat
@@ -14,11 +14,12 @@
 		if [ $hl_ver -ge 4 ]; then
 			echo tmp411 0x4d > /sys/bus/i2c/devices/i2c-1/new_device
 		fi
+		echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-1/new_device
 	;;
 	*)
+		echo powr1220 0x60 > /sys/bus/i2c/devices/i2c-1/new_device
 	;;
 	esac
-	echo powr1220 0x25 > /sys/bus/i2c/devices/i2c-1/new_device
 }
 
 function stop()
